packages/components/src/tooltip/index.js

```diff
@@ -212,9 +259,12 @@ function Tooltip( props ) {
 		? getDisabledElement
 		: getRegularElement;
 
+	const placement = positionToPlacement( position );
```

Comment:
Why do you think we need this here? Is the "positionToPlacement" in the "Popover" component not working properly? Should we fix that one instead?